src/oemof/solph/components/experimental/_link.py

```diff
@@ -81,14 +83,16 @@ def __init__(self, *args, **kwargs):
             for k, v in kwargs.get("conversion_factors", {}).items()
         }
 
-        wrong_args_message = (
-            "Component `Link` must have exactly"
-            + "2 inputs, 2 outputs, and 2"
-            + "conversion factors connecting these."
+        msg = (
+            "Component `Link` should have exactly "
+            + "2 inputs, 2 outputs, and 2 "
+            + "conversion factors connecting these. "
+            + "If this is intended and you know what you are doing you can "
+            + "disable the SuspiciousUsageWarning globally."
```

Comment:
The message does not state that the constructor is called deviating from this specification.

Reply:
Thanks for your review! Hope [4f058e4](https://github.com/oemof/oemof-solph/pull/827/commits/4f058e49d8ec33173b5c097d9507de0e7f68cdcc) helps.